### PR TITLE
When restarting test server with resetting the database, don't forget to close it

### PR DESCRIPTION
**Pull request by dsagal**

### CODE REVIEW

**Comment by fflorent** [comment]
Thanks @dsagal! This fixes the issue I get for my draft in #921

File: test/nbrowser/testServer.ts [hunk 1]
```diff
@@ -67,6 +67,9 @@ export class TestServerMerged extends EventEmitter implements IMochaServer {
     this._starts++;
     const workerIdText = process.env.MOCHA_WORKER_ID || '0';
     if (reset) {
+      // Make sure this test server doesn't keep using the DB that's about to disappear.
+      this.closeDatabase();
```
Comment (paulfitz) [inline test/nbrowser/testServer.ts:71]: Needs an await here.

Comment (dsagal) [reply test/nbrowser/testServer.ts]: Hmm, I guess lint config isn't set up to catch this in grist-core? Added await.

**Review by paulfitz — APPROVED** [review]
Thanks a lot for the fix @dsagal !

**Pull request merged**